src/app/components/annotations/pages/verification/verification.component.html

```diff
@@ -68,6 +68,23 @@ <h2>
     (focus)="verificationGridFocused = true"
     (blur)="verificationGridFocused = false"
   ></oe-verification>
+
+  @if (hasCorrectionTask()) {
+    <oe-verification
+      verified="skip"
+      shortcut="S"
+      (focus)="verificationGridFocused = true"
+      (blur)="verificationGridFocused = false"
+    ></oe-verification>
+
+    <oe-tag-prompt
+      #tagPrompt
+      verified="unsure"
+      shortcut="C"
+      (focus)="verificationGridFocused = true"
+      (blur)="verificationGridFocused = false"
+    ></oe-tag-prompt>
+  }
```

Comment:
ok discussed offline. there are some visual bugs, but:

skip should come last and should not be grouped with anything.  It also probably shouldn't be conditionally defined.